web/app/themes/hmcts-km/functions.php

```diff
@@ -32,7 +32,8 @@
 	'lib/users/author.php',          // Custom User Role Settings
 	'lib/users/editor.php',          // Custom User Role Settings
 	'lib/emails.php',                // Restricts certain email domain registration
-  'lib/security.php'
+  'lib/uploads.php',               // Added to extend allowed file types in Media upload
```

Comment:
I know this was already like this, but would be better practice if these were ordered alphabetically.